BiasAndFairnessServers/src/crud/bias_and_fairness.py

```
@@ -0,0 +1,43 @@
+
+async def upload_model(content, name, db):
+    result = db.execute(
+        text("INSERT INTO model_files (name, file_content) VALUES (:name, :file_content) RETURNING id"),
+        {"name": name, "file_content": content}
+    )
+    row = result.fetchone()
+    return row
+
+async def upload_data(content, name, target_column, sensitive_column, model_id, db):
+    result = db.execute(
+        text("INSERT INTO model_data (name, file_content, target_column, sensitive_column, model_id) VALUES (:name, :file_content, :target_column, :sensitive_column, :model_id) RETURNING id"),
```

Pylint (3.3.7)

[convention] Line too long (192/100)

(C0301)